embedding/cpp/elm_flutter_view.cc

```diff
@@ -12,6 +12,8 @@
 ElmFlutterView::~ElmFlutterView() {
   if (view_) {
     FlutterDesktopViewDestroy(view_);
+    engine_ = nullptr;
```

Comment:
Is this a necessary? As ElmFlutterView is deleted, engine_ will deleted.

Reply:
You're right. I was just trying to make this consistent with `ElmFlutterView.Destroy` (C#) and

https://github.com/flutter-tizen/flutter-tizen/blob/8d6f09c254575b3e66b7d070e78adaec9fa18ed3/embedding/cpp/flutter_app.cc#L57-L63

and didn't notice that it was within the class destructor.